### PR TITLE
Feature/ipte 22 export h2 database to sql script

### PR DESCRIPTION
**Pull request by Death111**

### CODE REVIEW

**Comment by Death111** [comment]
Hint: When creating new release, we need to include steps how to move old h2 database to new folder '/db/x.x.x/'

I need to update ReleaseNotes to include changes:
- database path includes database version
- Added export button in settings dialog, to export to .sql (for later import coming with next release)
- Logging build related infos on startup

**Review by JSchraff — COMMENTED** [review]
I see some Issues concerning the Import of the generated sql, see comment in SettingsController.

Even after Drop Statements is the execution of the  flyway migration after import necessary.
This might run into issues if a flyway migration was run inbetween export and Import if tables got renamed and if run again table already exists (changed tables will not be dropped).

**Review by JSchraff — APPROVED** [review]

File: src/main/java/de/doubleslash/keeptime/view/SettingsController.java [hunk 1]
```diff
@@ -1,334 +1,334 @@
-// Copyright 2019 doubleSlash Net Business GmbH
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/SettingsController.java:1]: you reformatted the whole file content

Comment (Death111) [reply src/main/java/de/doubleslash/keeptime/view/SettingsController.java]: fixed

**Pull request merged**